### PR TITLE
Open beta join and get started page

### PR DESCRIPTION
**Pull request by pietrorosa77**
Adding the new join open beta page and get started page.
Connected to save stats back end 

image
image
image
image

https://sourcegraph.atlassian.net/browse/CLOUD-300
https://sourcegraph.atlassian.net/browse/CLOUD-299
https://sourcegraph.atlassian.net/browse/CLOUD-312

## Test plan

code behind feature flag and tested manually. All code related to ux

### CODE REVIEW

File: client/web/src/org/openBeta/NewOrganizationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+                        <Checkbox
+                            name="userSearchable"
+                            id="userSearchable"
+                            value="searchable"
+                            checked={termsAccepted}
+                            required={true}
+                            onChange={onTermsAcceptedChange}
+                            label={
+                                <>
+                                    I accept the <Link to="/">terms of service</Link> for participating in the
```
Comment (kopancek) [inline client/web/src/org/openBeta/NewOrganizationPage.tsx:269]: Which link should this point to?

File: cmd/frontend/graphqlbackend/schema.graphql [hunk 2]
```diff
@@ -5194,6 +5204,24 @@ type AutocompleteMemberSearchItem {
     inOrg: Boolean
 }
 
+"""
+Result user returned by invite members autocomplete search.
```
Comment (kopancek) [inline cmd/frontend/graphqlbackend/schema.graphql:5208]: Should we change the type description to something else here?

**Review by kopancek — COMMENTED** [review]
Looks good overall. Added some comments.

File: client/web/src/org/openBeta/GettingStarted.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                            to={`/organizations/${org.name}/settings/repositories`}
+                        />
+                        <Step
+                            label="Invite your teammates"
+                            completeCondition={membersCompleted}
+                            textMuted={membersCompleted}
+                            to={`/organizations/${org.name}/settings/members`}
+                        />
+                        <Step
+                            label="Search across Acmecorp’s code"
```
[the remark this answers is not in the document]
Comment (pietrorosa77) [reply client/web/src/org/openBeta/GettingStarted.tsx:103]: indeed

File: client/web/src/org/openBeta/NewOrganizationPage.tsx (hunk 1 above)
Comment (pietrorosa77) [reply client/web/src/org/openBeta/NewOrganizationPage.tsx:269]: waiting for reply from @quinnkeast

File: cmd/frontend/graphqlbackend/schema.graphql (hunk 2 above)
Comment (pietrorosa77) [reply cmd/frontend/graphqlbackend/schema.graphql:5208]: copy/pate I admit i'm guilty!!!!

File: client/web/src/org/openBeta/NewOrganizationPage.tsx (hunk 1 above)
Comment (quinnkeast) [reply client/web/src/org/openBeta/NewOrganizationPage.tsx:269]: @ryphil is coordinating this.

**Review by kopancek — APPROVED** [review]

File: client/web/src/org/openBeta/GettingStarted.tsx [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+                {linkText}
+            </Link>
+        </small>
+    )
+}
+
+const SEARCH_STATUS_RADIX = 'sgGetStartedSearchStep'
+const getSearchStepStatus = (orgName: string): 'complete' | 'incomplete' | null => {
+    try {
+        return localStorage.getItem(`${SEARCH_STATUS_RADIX}${orgName}`) as 'complete' | 'incomplete' | null
```
Comment (limitedmage) [inline client/web/src/org/openBeta/GettingStarted.tsx:167]: Does this need to be localStorage, or can it use Temporary Settings instead? (eg. should the user go through this flow again if they log in to a different machine?)

**Pull request merged**